index.html

```diff
@@ -3,134 +3,401 @@
   // FETCH DATA
+  d3.json('data.json', function (error, json) {
+    if (error) { throw error; }
+    data = json;
+    setup();
+    update();
+  });
+  
 
-  function setup() {
-    // SVG
+  function setup() {    
 
+    // SVG 1
     svg = d3.select('#chart')
-      .append('svg');
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
-    // SCALES
+    //SVG 2
+     svg2 = d3.select('#chart2')
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+    
+    // SCALES 1
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.PovertyRate; })])
+      .range([0, width])
+      .nice();
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.rate; }))
+      .range([height, 0])
+      .nice();
+
+    rscale = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) {return d.Pop; })])
+      .range([2,20]);
+
+     // SCALES 2
+    x2 = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) { return d.MedianIncome; })])
+      .range([0, width])
+      .nice();
+    y2= d3.scale.linear()
+      .domain(d3.extent(data, function (d) { return d.rate; }))
+      .range([height, 0])
+      .nice();
+
+     
 
     // AXES
+    var padding=0
+    var axispadding=60
+    var xaxispadding=35
 
-    // EVENT HANDLERS
 
+    //Chart 1 Axes
+    var xAxis = d3.svg.axis()
+      .scale(x)
+      .orient("bottom")
+      .ticks(10);
+
+    var yAxis = d3.svg.axis()
+      .scale(y)
+      .orient("left")
+      .ticks(10);
+    
+    svg.append("g")
+      .attr("class","axis")
+      .attr("transform","translate(0,"+ height + ")")
+      .call(xAxis);
+
+    svg.append("g")
+      .attr("class","axis")
+      .attr("transform","translate("+ padding + ",0)")
+      .call(yAxis);
+
+    svg.append("text")
+      .attr("text-anchor", "middle")  // this makes it easy to centre the text as the transform is applied to the anchor
+      .attr("class","axislabel")
+      .attr("transform", "translate("+ (-axispadding) +","+(height/2)+")rotate(-90)")  // text is drawn off the screen top left, move down and out and rotate
+      .text("Arrests per 100,000 people");
+
+  svg.append("text")
+      .attr("text-anchor", "middle")  // this makes it easy to centre the text as the transform is applied to the anchor
+      .attr("class","axislabel")
+      .attr("transform", "translate("+ (width/2) +","+(height+(xaxispadding))+")")  // centre below axis
+      .text("Poverty Rate (%)");
+
+    //chart2 Axes
+
+    var xAxis2 = d3.svg.axis()
+      .scale(x2)
+      .orient("bottom")
+      .ticks(10);
```

Comment:
Wonder if you might use `.tickFormat()` here? Seems like probably could omit the ,000, maybe prepend a $ sign?